inginious/frontend/plugins/multilang/problems/code_multiple_languages_problem.py

```diff
@@ -5,6 +5,7 @@
 
 path_to_plugin = os.path.abspath(os.path.dirname(__file__))
 python_tutor_url = "http://127.0.0.1:8003/"
+linter_url = "http://localhost:4567/"
```

Comment:
Not sure how to asks for the url and port variables from the configuration.yaml file. Just copy the style of python_tutor_url

Reply:
I'll fix both